apps/utr-backend/docker-compose.yml

```diff
@@ -7,3 +7,13 @@ services:
       POSTGRES_DB: $DB_NAME
     ports:
       - "127.0.0.1:5433:5432"
+  utr-backend:
+    build:
+      context: ../../
+      dockerfile: ./apps/utr-backend/Dockerfile
+    environment:
+      HOST: 0.0.0.0
+      PORT: 8103
+      DATABASE_URL: postgresql://DB_USER:$DB_PWD@db:5432/$DB_NAME
```

Comment:
Bei uns läuft der db service in der docker compose ja unter Port `5433` siehe hier `"127.0.0.1:5433:5432"`.
Hier ist aber trotzdem `5432` richtig, da es nicht darum geht auf welchem Port sie lokal gemappt ist oder?
Ansonsten muss der auf `5433` gesetzt werden.

Reply:
Ah, genau. Docker baut sozusagen ein internes Netzwerk auf, in dem Postgres standardmäßig den Port 5432 hat. "127.0.0.1:5433:5432" sorgt dafür, dass der Port 5433 im lokalen Netzwerk auf 5432 im Docker-Netzwerk weitergeleitet wird, intern hat Postgres aber immernoch die 5432.